lib/postinstall.js

```diff
@@ -46,80 +49,76 @@ const writePrettierRc = (new Promise((resolve, reject) => {
 Make sure that it includes the following for 'eslint-config-motley'
 to work as it should:
 
-${JSON.stringify(content, null, 2)}`);
+${JSON.stringify(content, null, 2)}\n`);
 
-    return resolve();
+   return resolve();
   };
 
 
-  return fs.writeFile(prettierPath, JSON.stringify(content, null, 2), 'utf-8', (err) => {
+  fs.writeFile(prettierPath, `${JSON.stringify(content, null, 2)}\n`, 'utf-8', (err) => {
     if (err) {
-      return reject(err);
+     return reject(err);
     }
 
-    return resolve();
+   return resolve();
   });
 }));
 
-const writeToFile = (packageJsonPath, jsonString) => fs.writeFile(packageJsonPath, jsonString, 'utf-8', (err) => {
-  if (err) {
-    return reject(err);
-  }
-
-  return resolve();
-});
```

Comment:
This extracted function caused trouble when being called from `writeToPackageJson`. `reject` and `resolve` aren't defined here! Moved it back inline since it's only used once there anyway.

Reply:
The implementation had a bug where we didn't write to file at all if `lint-staged` was specified despite other stuff missing.

The actual fuckup in #55 was that I didn't promisify the callback function (which I could have done). When this was made Node didn't have `util.promisify` at all.